src/olympia/api/jwt_auth.py

```diff
@@ -96,7 +96,7 @@ def jwt_decode_handler(token, get_api_key=APIKey.get_jwt_key):
             '{e.__class__.__name__}: {e}'.format(e=exc)
         )
         raise exceptions.AuthenticationFailed(detail=f'Invalid JWT: {exc}.')
-    except jwt.InvalidIssuedAtError as exc:
+    except (jwt.exceptions.ImmatureSignatureError, jwt.InvalidIssuedAtError) as exc:
```

Comment:
What's an immature signature? Is that like how my son signs his name? 😛 

But seriously, do we have test coverage for this, and if so, should the test be updated to check both exceptions?

Reply:
Immature as in not old enough - the time is in the future 😉 

https://github.com/mozilla/addons-server/blob/master/src/olympia/api/tests/test_jwt_auth.py#L85:L98 tests for this scenario.  (Clock skew was previously just thrown as `InvalidIssuedAtError`, but now that's just thrown when the value isn't a number.)  The test checks for the exception we raise from the function instead - `AuthenticationFailed` and the error string. I didn't think it worthwhile to update the text we return as it was the same cause as before.